resources/scripts/components/server/backups/BackupContainer.tsx

```diff
@@ -17,6 +17,7 @@ export default () => {
 
     const backups = ServerContext.useStoreState(state => state.backups.data);
     const setBackups = ServerContext.useStoreActions(actions => actions.backups.setBackups);
+    const server = ServerContext.useStoreState(state => state.server.data!);
```

Comment:
`const { name } = useServer()` is my new method of doing this for the sake of readability and not endlessly typing the same stuff over and over.

I also generally attempt to keep these in some sort of length based order/grouping by what they're doing (but its kinda arbitrary, so don't worry too much).